src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -643,4 +643,37 @@
   <data name="PX1072Title_CreateSharedGraphVariable" xml:space="preserve">
     <value>Creating a graph for a BQL query is a resource consuming operation. To decrease the number of these operations, create a variable with a shared graph that can be reused by BQL queries.</value>
   </data>
+  <data name="PX1062MessageFormat" xml:space="preserve">
+    <value>Graphs and graph extensions cannot declare static {0}.</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Fields" xml:space="preserve">
+    <value>mutable fields</value>
+  </data>
+  <data name="PX1062MessageFormatArg_Properties" xml:space="preserve">
+    <value>mutable properties</value>
+  </data>
+  <data name="PX1062FixMakeReadOnlyFormat" xml:space="preserve">
+    <value>Make the {0} read-only</value>
```

Comment:
@EAndrosova  please review the code fix message: `Make the {0} read-only`
The possible format arguments (stored here in the resources):
- field
- property
- view
- action

Reply:
To follow some grammar rules, please change to "Add the readonly modifier to the field" or "Make the field read-only."

Reply:
The current code fix message is exactly like suggested except the dot in the end. 
I think we should not place dots at the end of diagnostic and code fix messages, I explained the reasons in this comment:
https://github.com/Acumatica/Acuminator/pull/462#issuecomment-1306173979

Reply:
After discussion it was decided to follow MS style for  diagnostic messages:

- No dots for single sentences
- Dots at the end of all sentences if there are two or more sentences